content/news/playtest-20150719.md

```diff
@@ -0,0 +1,46 @@
+We are pleased to announce the second playtest on the road to our next stable release.  With this playtest, we have entered the feature freeze phase, so it already is pretty close to what the release will eventually look like. There are probably a number of bugs still lurking in the code, so it needs testing, testing and some more testing!
+
+Compared to the previous playtest, there have been a number of new features, fixes and other changes all over the place. Music playback has received quite a number of changes. Maps can now define a soundtrack, which defaults to a random track, and also separate soundtracks for victory and defeat which are played on the score screen.
```

Comment:
This is a really long sentence.  How about "Maps now have much greater control over their music, including default (starting) music and custom victory and defeat tracks".